java/ql/integration-tests/java/query-suite/test.py

```diff
@@ -0,0 +1,15 @@
+import runs_on
+
+@runs_on.linux
+def test(codeql, java, cwd, expected_files, semmle_code_dir):
+    query_suites = ['java-code-quality.qls', 'java-security-and-quality.qls', 'java-security-extended.qls', 'java-code-scanning.qls']
+
+    for query_suite in query_suites:
```

Comment:
small nit: this looks like a parametrized test, so you can maybe write it as one:
```suggestion
import pytest

@runs_on.linux
@pytest.mark.parametrize("query_suite", ['java-code-quality.qls', 'java-security-and-quality.qls', 'java-security-extended.qls', 'java-code-scanning.qls'])
def test(codeql, java, cwd, expected_files, semmle_code_dir, query_suite):
```
This will be parallelizable a bit better, but not by a great deal, so feel free to ignore this.